JSDemos/Demos/Popup/Overview/Angular/description.md

```diff
@@ -0,0 +1,12 @@
+The **Popup** UI component is a pop-up window that overlays the current view and displays specified content.
+
+This demo illustrates the following **Popup** use cases:    
+
+- **Specify the content**       
+Define the **Popup** content within the UI component's markup.   
```

Comment:
Again, just trying to be a bit more conversational - trying not to sound too technical (use simple words). 

```suggestion
Put any valid markup inside the component. 
```